parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -5049,3 +5051,48 @@ SqlCloseCursor SqlCloseCursor() :
     <CLOSE> cursorName = SimpleIdentifier()
     { return new SqlCloseCursor(s.end(this), cursorName); }
 }
+
+SqlSelectAndConsume SqlSelectAndConsume() :
+{
+    boolean set = false;
+    final List<SqlNode> selectList;
+    final SqlNodeList parameters = new SqlNodeList(getPos());
+    final SqlIdentifier fromTable;
+    final int topNum;
+    final Span s = Span.of();
+    SqlNode e;
+}
+{
+    <SELECT>
```

Comment:
The docs state that <SEL> is also valid here.

Reply:
Yeah you're right, I was looking at a newer version of the docs and I think it has a typo? It says "SET" is allowed instead of "SEL", but I think SEL would make more sense here